### DISCUSSION TITLE
Дропаут (удалить случайные)

### DISCUSSION THREAD

**Discussion by kisoid** [Chimildic/YaMuTools#53] [Идея] [answered]
Предлагаю добавить в раздел "Фильтр треков" новую функцию "Удалить случайные"

В качестве аргумента - % удаляемых треков (лучше именно %, а не кол-во треков, т.к. в веб-версии кол-во треков в плейлисте не известно)

То есть, если у меня плейлист на 1000 треков, и я выбрал удаление 30% случайных, то должно остаться 700 треков.
Очень важно, чтобы порядок следования оставшихся треков не поменялся.

UPD
Похожий запрос: https://github.com/Chimildic/YaMuTools/discussions/33

Кейс использования такой же - регулировка размера плейлиста.
Например, после всех фильтраций (недавно игравшие, дубликаты и т.д.) получился плейлист на 100 часов, а хочется уменьшить его до 10 часов (чтобы есть слона частями)

**Comment by Chimildic** [comment, marked as answer]
Добавил в следующее обновление, ожидайте
